modules/kargoBidAdapter.js

```diff
@@ -95,6 +95,7 @@ function buildRequests(validBidRequests, bidderRequest) {
     },
     imp: impressions,
     user: getUserIds(tdidAdapter, bidderRequest.uspConsent, bidderRequest.gdprConsent, firstBidRequest.userIdAsEids, bidderRequest.gppConsent),
+    site: firstBidRequest.ortb2.site
```

Comment:
Might need to add `null` checks for `ortb2`:
```
select
count(time_id_utc) as total
from stage_kraken_bid_request e
where time_id_utc = 2023100310
and e.src:data:body:bidRequest:prebidRawBidRequests[0] is not null
and e.src:data:body:bidRequest:prebidRawBidRequests[0]:ortb2 is null;

---------------------------------------------

TOTAL
33,079
```

```
select
count(time_id_utc) as total
from stage_kraken_bid_request e
where time_id_utc = 2023100310
and e.src:data:body:bidRequest:prebidRawBidRequests[0]:ortb2 is not null
and e.src:data:body:bidRequest:prebidRawBidRequests[0]:ortb2:site is null;

------------------------------------------------

TOTAL
0
```

Reply:
Updated